framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -792,17 +796,35 @@ describe('BaseCrossChainUpdateCommand', () => {
 			await expect(
 				command['beforeCrossChainMessagesExecution'](executeContext, true),
 			).resolves.toEqual([[], false]);
-			expect(internalMethod.terminateChainInternal).toHaveBeenCalledWith(
-				expect.anything(),
-				params.sendingChainID,
+
+			expect(command['verifyRoutingRules']).toHaveBeenCalledWith(
+				ccm,
+				executeContext.params,
+				executeContext.chainID,
+				true,
 			);
 
-			const ccmID = getIDFromCCMBytes(codec.encode(ccmSchema, ccm));
-			const ccmEventQueue = executeContext.eventQueue.getChildQueue(
-				Buffer.concat([EVENT_TOPIC_CCM_EXECUTION, ccmID]),
+			// let's make sure it indeed throws expected error
+			expect(() => {
+				command['verifyRoutingRules'](ccm, executeContext.params, executeContext.chainID, true);
+			}).toThrow('blah');
```

Comment:
This looks unnecessary as in the same test we are mocking the implementation to throw error, with this approach we will always have to add an extra expectation wherever we are mocking the implementation